_studio/mfx_lib/encode_hw/shared/ehw_task_manager.cpp

```diff
@@ -208,6 +212,10 @@ mfxStatus TaskManager::TaskQuery(StorageW& inTask)
 
     MoveTask(Stage(S_QUERY), Stage(S_NEW), FixedTask(*pTask));
 
+    if (m_nRecodeTasks)
+    {
+        m_bSync = true; return MFX_TASK_WORKING;
```

Comment:
move return to new line, add comment why we need to repeat async cycle

Reply:
Fixed